server/sonar-web/src/main/js/apps/account/organizations/OrganizationCard.js

```diff
@@ -52,11 +53,22 @@ export default function OrganizationCard(props /*: Props */) {
         <OrganizationLink organization={organization}>{organization.name}</OrganizationLink>
       </h3>
 
-      <div className="account-project-key">{organization.key}</div>
-
       {!!organization.description && (
         <div className="account-project-description">{organization.description}</div>
       )}
+
+      <div className="account-project-key">
+        <span className="little-spacer-right">
+          {translate('key')}
+          {':'}
+        </span>
+        <input
+          onClick={event => event.currentTarget.select()}
```

Comment:
Maybe extract that function, but I don't think it's really important on that page

Reply:
since `input` is not a pure component, it's fine to pass new function every time :) I hope :)